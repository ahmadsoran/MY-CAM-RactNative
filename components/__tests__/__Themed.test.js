import React from "react";
import renderer from "react-test-renderer";
import { View } from "../Themed";

describe("<View />", () => {
  it("has 1 child", () => {
    const tree = renderer.create(<View />).toJSON();
    expect(tree).toMatchSnapshot({
      children: 1,
      props: {
        darkColor: "",
      },
    });
  });
});
