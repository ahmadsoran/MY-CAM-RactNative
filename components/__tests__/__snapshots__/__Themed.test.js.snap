// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<View /> has 1 child 1`] = `
<View
  style={
    Array [
      Object {
        "backgroundColor": "#fff",
        "direction": "ltr",
      },
      undefined,
    ]
  }
/>
`;
